From 32ba6b8bf5d5f6d62e558965308e93e13d95d76f Mon Sep 17 00:00:00 2001
From: Martchus <martchus@gmx.net>
Date: Sun, 25 Sep 2016 21:53:59 +0200
Subject: [PATCH 1/9] Use correct ICU libs

The ICU libraries used for cross-compilation are named
exactly the same as their native Linux counterpart
---
 Source/WTF/WTF.pri | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/WTF/WTF.pri b/Source/WTF/WTF.pri
index 1f4866d66..5d8933ae1 100644
--- a/Source/WTF/WTF.pri
+++ b/Source/WTF/WTF.pri
@@ -14,7 +14,7 @@ mac {
     INCLUDEPATH = $${ROOT_WEBKIT_DIR}/Source/WTF/icu $$INCLUDEPATH
     LIBS += -licucore
 } else:!use?(wchar_unicode): {
-    win32 {
+    win32:!mingw {
         CONFIG(static, static|shared) {
             CONFIG(debug, debug|release) {
                 LIBS += -lsicuind -lsicuucd -lsicudtd
-- 
2.11.0

